### ISSUE TITLE
Doesn't adapt to device orientation change

### ISSUE THREAD

**Issue by mitchray** [dimsemenov/PhotoSwipe#831]
I created a demo based on the PhotoSwipe homepage: http://codepen.io/mitchray/full/NqqZpV/

Steps to reproduce:
1. Click a photo to open it in the fullscreen viewer.
2. Change device orientation twice (eg from portait, to landscape and back to portrait)

Result:
![2015-05-04_17-37-22](https://cloud.githubusercontent.com/assets/5735900/7450098/4e4406dc-f284-11e4-9588-0e583c915b3d.png)

Happens regardless of history true or false.

**Comment by dimsemenov** [comment]
Device? OS? Browser?

**Comment by mitchray** [comment]
Apologies for leaving that out.

Win 7 Chrome 42.0.2311.135
Win 7 Opera 26.0.1656.32
Android 4.2.2 Chrome 42.0.2311.109
Android 4.2.2 Opera 29.0.1809.91837

Latest versions of Firefox on both platforms is fine.

**Comment by dimsemenov** [comment]
Thanks. Please also elaborate what exactly device do you use. it's important when it comes to Android...

Also, what's the device on Win7?

**Comment by mitchray** [comment]
Win 7 Desktop PC
Android is a Galaxy S2

**Comment by dimsemenov** [comment]
How do you reproduce the issue on desktop PC? You just resize browser window, or use Chrome dev tools to emulate orientation and device viewport?

**Comment by mitchray** [comment]
Yep, both methods produce the same result.

**Comment by dimsemenov** [comment]
I was able to replicate. It's just because you haven't closed `.row` element, which places `.pswp` element inside row, which gives you gap from top. 

Can you repeat here: http://codepen.io/dimsemenov/pen/LVVwjq ?

**Comment by mitchray** [comment]
Your pen fixed the demo I posted, but it turns out that wasn't the exact issue I was having on my site. However you did lead me to find the real culprit!

The problem was the extra whitespace between the last image and its container. In my demo that was caused by the open tag (my bad copy/pasting), and on my site it was caused by the several placeholder elements I have (its an inline-block grid of images).

Thankyou for investigating my issue and ending my 2 days of going crazy.
